src/v0/destinations/kustomer/transform.js

```diff
@@ -85,6 +85,11 @@ const constructKustomerPayload = (message, category, email) => {
 // type of events.
 const responseBuilderSimple = async (message, category, destination) => {
   let payload = {};
+
+  // Reference for base endpoint
+  // https://developer.kustomer.com/kustomer-api-docs/reference/getting-started-with-kustomer-api#using-the-kustomer-api
```

Comment:
can we add comment on what options would we get from `destination.Config` for easy readability ? Otherwise, one has to go to integ-config project to know the options.